processing/src/main/java/io/druid/query/select/SelectQueryQueryToolChest.java

```diff
@@ -247,6 +249,16 @@ public Object apply(final Result<SelectResultValue> input)
                         {
                         }
                     ),
+                    (Set<String>) jsonMapper.convertValue(
+                        resultIter.next(), new TypeReference<Set<String>>()
+                        {
+                        }
+                    ),
+                    (Set<String>) jsonMapper.convertValue(
+                        resultIter.next(), new TypeReference<Set<String>>()
+                        {
+                        }
+                    ),
```

Comment:
Since the layout of the cached object is changed here, We need to change the cacheKey also, otherwise the queries which were already cached will start failing after upgrade.
Probably upgrading the byte for SELECT_QUERY would be fine.